prpr_plugin

pokemonchw opened the pull request:
A plugin for mimics kotomei_bot

pokemonchw committed:
一个仿kotomei_bot的舔bot插件

Z4HD reviewed (changes requested):
建议使用docstring（文档字符串）在`prpr.py`中描述一下插件的用途。

Commit:
Update alipay_redpack.py

Z4HD committed:
Release v3.3 (#43) from jqqqqqqqqqq/dev

- New configuration item `USE_SHORT_URL`: You can configure whether to use short links in the configuration file.
- JSON configuration file now supports optional configuration
- Support for using docker-compose orchestration services
- Messages beginning with "//" will not be forwarded
- Partial details optimization
- Management functions are still under development
- Added some plugin documentation.
- Bug fixs

pokemonchw committed:
Merge pull request #1 from jqqqqqqqqqq/master

fork main

pokemonchw committed:
地球监测站版本同步

jqqqqqqqqqq commented:
20000 addition，这到底是加了啥（

jqqqqqqqqqq commented:
功能改动与主版本有冲突
附带的插件我在考虑写一个更好的插件逻辑以及提供第三方插件部署方案

所以，先等我有时间写完新的插件逻辑再考虑部分 merge
